js/core/src/reflection.ts

```diff
@@ -226,10 +226,9 @@ export class ReflectionServer {
 
     server.post('/api/notify', async (request, response) => {
       const { telemetryServerUrl, reflectionApiSpecVersion } = request.body;
-      if (typeof telemetryServerUrl === 'string') {
-        setTelemetryServerUrl(telemetryServerUrl);
-        logger.debug(`Connected to telemetry server on ${telemetryServerUrl}`);
-      }
+      const url = process.env.GENKIT_TELEMETRY_SERVER ?? telemetryServerUrl;
```

Comment:
Kinda funky logic here, we should just return from this method early if the env var is set. No reason to call `setTelemetryServerUrl` every time that notify gets hit in the env var scenario.

Reply:
The original reason why I had this in is because the app needs at least one `setTelemetryServerUrl` to properly configure TS. In light of @pavelgj's earlier comment to add ta call to tracing.ts, the original reason no longer holds, is that correct?

Reply:
Right! Better to handle the env var scenario there.